CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Dradis Framework 3.16 (XXXX, 2020) ##
+
+* Use the new built-in Title tag rather than the Type tag
```

Comment:
Do we know the version of netsparker that added the title, as this would not be a backwards compatible change by itself it could be worth noting in the changelog where we dropped compatibility.

Reply:
We don't and I don't think it's worth chasing Netsparker support about it.